src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1133,7 +1133,22 @@ private uint getClassSize(CORINFO_CLASS_STRUCT_* cls)
         }
 
         private uint getClassAlignmentRequirement(CORINFO_CLASS_STRUCT_* cls, bool fDoubleAlignHint)
-        { throw new NotImplementedException("getClassAlignmentRequirement"); }
+        {
+            if (_compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.ARM ||
+                _compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.ARMEL)
+            {
+                // Default alignment is sizeof(void*)
+                uint result = (uint)sizeof(void*);
```

Comment:
`sizeof(void*)` is the size of the pointer of the host where the compiler is running. Instead, this should be `Target.PointerSize` - size of the pointer on the target.